desktop/apps/editorial_features/components/venice_2017/stylesheets/body.styl

```diff
@@ -102,6 +103,40 @@
       &:hover
         background-color white
         color black
+  &__follow
+    p
+      garamond-size('l-body')
+      margin-bottom 10px
+  &__follow-item
+    clear both
+    display none
+  .artist-follow
+    display flex
+    align-items center
+    float left
+    margin-left -5px
+    .follow-text
+      display flex
+      align-items center
+      &:before
+        font-family artsy-icons
+        font-size 32px
+        line-height 32px
+      &:after
+        margin-right 4px
+    &[data-state=follow]
+      .follow-text:after
+        content 'Follow'
+      .follow-text:before
+        content '\e629'
+    &[data-state=following]
+      .follow-text:after
+        content 'Following'
+      .follow-text:before
+        content '\e628'
+    &[data-state=following]:hover
+      .follow-text:after
+        content 'Unfollow'
```

Comment:
Wonder if we could just use the styling from  https://github.com/artsy/force/blob/master/desktop/components/follow_button/index.styl ?